content/en/internal/_index.md

```diff
@@ -6,6 +6,7 @@ type: "docs"
 menu:
   main:
     weight: 30
+    pre: <i class='fa fa-cogs'></i>
```

Comment:
I feel this is making things more complex/difficult to read, especially as the icon is really close to the text, I don't feel like it works well with the current setup:
![Screenshot 2021-07-12 at 14 54 51](https://user-images.githubusercontent.com/451417/125290945-250d8100-e321-11eb-90fe-80bd83103c34.png)

Reply:
It's easy to remove.

Reply:
But on my browsers looks better. 

Opera:
![image](https://user-images.githubusercontent.com/25453173/125291679-f348ea00-e321-11eb-85ac-a8cec7eb6ff0.png)

Edge (the new Chromium-based one):
![image](https://user-images.githubusercontent.com/25453173/125291795-16739980-e322-11eb-8045-7a2962a0c4b9.png)

Firefox:
![image](https://user-images.githubusercontent.com/25453173/125291918-35722b80-e322-11eb-88a1-644cf54d1084.png)

Reply:
So the theme wasn't properly updated in my first checkout, no that's fine.
Yes it's a lot more clear, thanks!
![Screenshot 2021-07-12 at 15 28 21](https://user-images.githubusercontent.com/451417/125295735-d0203980-e325-11eb-9145-0cdea2920067.png)

Let's keep it and see how it goes.